core/src/main/java/org/apache/carbondata/core/reader/CarbonDictionaryMetadataReaderImpl.java

```diff
@@ -128,6 +131,9 @@ public CarbonDictionaryMetadataReaderImpl(
         break;
       }
     }
+    if (null == dictionaryChunkMeta) {
+      throw new IOException("Last dictionary chunk does not exist");
```

Comment:
Message should be "Matching dictionary chunk for offset not found"

Reply:
fixed